spec/index.html

```diff
@@ -393,11 +398,11 @@ <h2>Canonicalization State</h2>
 
     <dl>
       <dt><dfn>blank node to quads map</dfn></dt>
-      <dd>A data structure that maps a <a>blank node identifier</a> to
+      <dd>A <a data-cite="INFRA#ordered-map">map</a> that relates a <a>blank node identifier</a> to
         the <a>quads</a> in which they appear in the
         <a>input dataset</a>.</dd>
       <dt><dfn>hash to blank nodes map</dfn></dt>
-      <dd>A data structure that maps a <a>hash</a> to a list of
+      <dd>A <a data-cite="INFRA#ordered-map">map</a> that relates a <a>hash</a> to a list of
```

Comment:
```suggestion
      <dd>An <a data-cite="INFRA#ordered-map">ordered map</a> that relates a <a>hash</a> to a list of
```

Reply:
I don’t think it was ordered before, is order significant here as well? Of course, the citation is to an ordered map in any case.

Reply:
Oh, if it wasn't ordered before, nevermind. I may have mixed this up with the identifier issuer map.